Description: Change README.md file
 Replace first line of README.md file with correct package name.
Author: Sagar Ippalpalli <i.vikram15@gmail.com>
#Origin: <upstream|backport|vendor|other>, <URL, required except if Author is present>
Forwarded: no
#Applied-Upstream: <version|URL|commit, identifies patches merged upstream, optional>
#Reviewed-by: <name and email of a reviewer, optional>
#Last-Update: <YYYY-MM-DD, last update of the meta-information, optional>
---
#This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/README.md
+++ b/README.md
@@ -1,4 +1,4 @@
-# rails-asset-jqgrid
+# rails-asset-jqueryui
 
 Gemified/Assetized version of the jqueryui library complete with themes!
 
